crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -536,35 +549,212 @@ pub enum AuthorizedotnetVoidStatus {
+                        field_name: "connector_metadata",
+                    })?;
+                let url = transaction_response
+                    .secure_acceptance
+                    .as_ref()
+                    .and_then(|x| x.secure_acceptance_url.to_owned());
+                let redirection_data =
+                    url.map(|url| services::RedirectForm::from((url, services::Method::Get)));
+                Ok(Self {
+                    status,
+                    response: match error {
+                        Some(err) => Err(err),
+                        None => Ok(types::PaymentsResponseData::TransactionResponse {
+                            resource_id: types::ResponseId::ConnectorTransactionId(
+                                transaction_response.transaction_id.clone(),
+                            ),
+                            redirection_data,
+                            mandate_reference: None,
+                            connector_metadata: metadata,
+                            network_txn_id: transaction_response
+                                .network_trans_id
+                                .clone()
+                                .map(|network_trans_id| network_trans_id.expose()),
+                            connector_response_reference_id: Some(
+                                transaction_response.transaction_id.clone(),
+                            ),
+                            incremental_authorization_allowed: None,
+                        }),
+                    },
+                    ..item.data
+                })
+            }
+            Some(TransactionResponse::AuthorizedotnetTransactionResponseError(_)) | None => {
+                Ok(Self {
+                    status: enums::AttemptStatus::Failure,
+                    response: Err(get_err_response(item.http_code, item.response.messages)?),
+                    ..item.data
+                })
+            }
+        }
+    }
+}
+
+impl<F>
+    TryFrom<
+        types::ResponseRouterData<
+            F,
+            AuthorizedotnetPaymentsResponse,
+            types::CompleteAuthorizeData,
+            types::PaymentsResponseData,
+        >,
+    > for types::RouterData<F, types::CompleteAuthorizeData, types::PaymentsResponseData>
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: types::ResponseRouterData<
+            F,
+            AuthorizedotnetPaymentsResponse,
+            types::CompleteAuthorizeData,
+            types::PaymentsResponseData,
+        >,
+    ) -> Result<Self, Self::Error> {
+        match &item.response.transaction_response {
+            Some(TransactionResponse::AuthorizedotnetTransactionResponse(transaction_response)) => {
+                let status = enums::AttemptStatus::foreign_from((
+                    transaction_response.response_code.clone(),
+                    true,
```

Comment:
Please confirm this whether wallets support manual capture or not

Reply:
I have added is_auto_capture for CompleteAuthorize also